Maybe this can be entirely eliminated with new maven args: -P no-git-rev -Dgit.commit.id.describe=${DISTVERSION} 

--- pom.xml.orig	2019-07-22 05:00:43 UTC
+++ pom.xml
@@ -133,25 +133,6 @@
             </plugin>
 
             <plugin>
-                <groupId>pl.project13.maven</groupId>
-                <artifactId>git-commit-id-plugin</artifactId>
-                <version>2.2.1</version>
-                <executions>
-                    <execution>
-                        <id>get-the-git-infos</id>
-                        <goals>
-                            <goal>revision</goal>
-                        </goals>
-                    </execution>
-                </executions>
-                <configuration>
-                    <dotGitDirectory>${project.basedir}/.git</dotGitDirectory>
-                    <verbose>false</verbose>
-                    <commitIdGenerationMode>flat</commitIdGenerationMode>
-                </configuration>
-            </plugin>
-
-            <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-checkstyle-plugin</artifactId>
                 <version>2.17</version>
@@ -330,4 +311,4 @@
         <connection>scm:git:file://localhost/${pom.basedir}</connection>
     </scm>
 
-</project>
\ No newline at end of file
+</project>
